### PR TITLE
Add array operators to README

### PR DESCRIPTION
**Pull request by bvsharp**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -348,20 +348,36 @@ Example:
 {"field1": {"$nin": ["value1", "value2"]}}
 ```
 
-### $contains
-Matches all records where the field (an array) contains the specified value. Supports string, number, and uuid types.
+### $contains_all
+Matches all records where the field (an array) contains all the specified values. Supports array of string, number, uuid, and boolean types.
```
Comment (ncovercash) [inline README.md:352]: nit
```suggestion
Matches all records where an array field contains all the specified values. Supports an array of string, number, uuid, or boolean types.
```

File: README.md [hunk 2]
```diff
@@ -348,20 +348,36 @@ Example:
...
 Example:
 ```json
-{"field1": {"$contains": 12}}
+{"field1": {"$contains_all": [1, 2, 3]}}
 ```
 
-### $not_contains
-Matches all records where the field (an array) does not contain the specified value, or is null. Supports string, number, and uuid types.
+### $not_contains_all
+Matches all records where the field (an array) does not contain all the specified values. Supports array of string, number, uuid, and boolean types.
```
Comment (ncovercash) [inline README.md:360]: nit
```suggestion
Matches all records where an array field does not contain all the specified values. Supports an array of string, number, uuid, or boolean types.
```

File: README.md [hunk 3]
```diff
@@ -348,20 +348,36 @@ Example:
...
 
 Example:
 ```json
-{"field1": {"$not_contains": "value1"}}
+{"field1": {"$not_contains_all": [1, 2, 3]}}
+```
+
+### $contains_any
+Matches all records where the field (an array) contains any of the specified values. Supports array of string, number, uuid, and boolean types.
```
Comment (ncovercash) [inline README.md:368]:
```suggestion
Matches all records where an array field contains any of the specified values. Supports an array of string, number, uuid, or boolean types.
```

File: README.md [hunk 4]
```diff
@@ -348,20 +348,36 @@ Example:
...
+
+Example:
+```json
+{"field1": {"$contains_any": [1, 2, 3]}}
+```
+
+### $not_contains_any
+Matches all records where the field (an array) does not contain any of the specified values. Supports array of string, number, uuid, and boolean types.
```
Comment (ncovercash) [inline README.md:376]:
```suggestion
Matches all records where an array field does not contain any of the specified values. Supports an array of string, number, uuid, or boolean types.
```

**Review by ncovercash — APPROVED** [review]
grammar nits

**Comment by kjain110** [comment]
This PR checks our English vocabulary :)

**Review by kjain110 — APPROVED** [review]

**Pull request merged**